src/app/clients/ActionBar.tsx

```diff
@@ -0,0 +1,188 @@
+"use client";
+
+import supabase, { Schema } from "@/supabase";
+import React, { useState } from "react";
+import ActionBarModal from "@/app/clients/ActionBarModal";
+import styled from "styled-components";
+import Button from "@mui/material/Button/Button";
+import Menu from "@mui/material/Menu/Menu";
+import MenuList from "@mui/material/MenuList/MenuList";
+import MenuItem from "@mui/material/MenuItem/MenuItem";
+import { Dayjs } from "dayjs";
+
+interface Props {
+    selected: number[];
+    data: Schema["parcels"][];
+}
+
+const statuses = [
+    "Request Denied",
+    // probably shouldn't be a status (?)
+    "No Status",
+    "Pending More Info",
+    "Called and Confirmed",
+    "Called and No Response",
+    "Ready to Dispatch",
+    "Received by Centre",
+    "Collection Failed",
+    "Parcel Collected",
+    "Delivered",
+    "Delivery Failed",
+    "Delivery Cancelled",
+    "Fulfilled with Trussell Trust",
+];
+
+const actions = [
+    "Generate Map",
+    "Print Day Overview",
+    "Print Shopping List",
+    "Print Shipping Label",
+    "Print Driver Overview",
+    "Delete Request",
+];
+
+const OuterDiv = styled.div`
+    display: flex;
+    padding: 1rem;
+    gap: 1rem;
+    border-radius: 0.5rem;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
+    margin: 1rem;
```

Comment:
Why use both padding and margin? Just cos surely `padding: 2rem` would have the same effect. Or is this just conventional to use both?

Reply:
They aren't the same with a `background-color` or `background-image`, the decoration is applied to the padding area but not the margin area.